make/compiler/Makefile.clang

```diff
@@ -59,11 +59,11 @@ OPT_CFLAGS = -O3
 #
 COMP_CFLAGS = $(CFLAGS)
 COMP_CFLAGS_NONCHPL = -Wno-error
-RUNTIME_CFLAGS = -std=c99 $(CFLAGS)
+RUNTIME_CFLAGS = -std=gnu99 $(CFLAGS)
```

Comment:
Yeah, I don't really understand why we set it, but we do this for GCC too... I believe it enables by default some functionality, but don't remember what issue I ran into here...